enable-and-configure-rbac.groovy

```diff
@@ -0,0 +1,333 @@
+import jenkins.model.Jenkins
+import hudson.security.Permission
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationStrategyImpl
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationConfig
+import nectar.plugins.rbac.strategy.RoleMatrixAuthorizationPlugin
+import nectar.plugins.rbac.groups.Group
+import nectar.plugins.rbac.groups.GroupContainer
+import nectar.plugins.rbac.groups.GroupContainerLocator
+import nectar.plugins.rbac.groups.RootProxyGroupContainer
+import nectar.plugins.rbac.roles.Role
+import nectar.plugins.rbac.roles.Roles
+import java.io.IOException
+import org.yaml.snakeyaml.Yaml
+/*
+ * Some important lessons I learned writing this code:
```

Comment:
Please remove the section lesson learned to keep it personally for your record

Reply:
That documentation is not the friendliest to understand, but I'll change my comment to link to it instead.

Reply:
But just keep it for yourself, please do no add that content to the script